scripts/vet.sh

```diff
@@ -184,4 +184,11 @@ revive -formatter plain ./... >"${REV_OUT}" || true
 # TODO: Remove `|| true` to unskip linter failures once existing issues are fixed.
 (noret_grep -v "unused-parameter" "${REV_OUT}" | not grep -v "\.pb\.go:") || true
 
+# Collection of trailing spaces analysis checks
+TRAIL_SPACE_OUT="$(mktemp)"
```

Comment:
Let's use a pipe instead of creating a file whenever possible, please.

Reply:
Done